ec2/spark_ec2.py

```diff
@@ -569,15 +569,34 @@ def launch_cluster(conn, opts, cluster_name):
         master_nodes = master_res.instances
         print "Launched master in %s, regid = %s" % (zone, master_res.id)
 
```

Comment:
Well, in this case I think the 5 seconds will just come out of the time we'll spend waiting anyway for SSH to become available.